### ISSUE TITLE
Missing cache in CacheReset

### ISSUE THREAD

**Issue by migo315** [bestit/contentful-symfony-bundle#13]
The `CacheResetService` only contains the `best_it_contentful.cache.pool.delivery` cache pool. But the updated Contentful entry could be in `best_it_contentful.cache.pool.routing` too, because the `ContentfulSlugMatcher` saves the complete entry result in `best_it_contentful.cache.pool.routing`. 

Currently, updated sites in Contentful will not receive the application.

**Comment by migo315** [comment]
@avareldev Does this issue is related to our problem today?

**Comment by b3nl** [comment]
> @avareldev Does this issue is related to our problem today?

Yes. But your customer is using the same cache for routing and delivery, so that is not the true problem for your problems 6 days ago. But nonetheless, this should be fixed generally!

**Comment by migo315** [comment]
ah ok
